sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -40,19 +39,24 @@ class ProjectOverviewView extends VerticalLayout{
     }
 
     private void fillGrid(){
-        projectGrid.addColumn({ it.projectId.projectCode.code})
-                .setCaption("Project Code").setId("ProjectCode")
-        projectGrid.addColumn({ it.projectTitle })
-                .setCaption("Project Title").setId("ProjectTitle").setWidth(GridUtils.TITLE_COLUMN_WIDTH)
+        projectGrid.addColumn({ it.code})
+                .setCaption("Project Code").setId("ProjectCode").setMaximumWidth(GridUtils.MAX_CODE_COLUMN_WIDTH)
+        projectGrid.addColumn({ it.title })
+                .setCaption("Project Title").setId("ProjectTitle")
+        projectGrid.addColumn({it.samplesReceived})
+                .setCaption("Samples Received").setId("SamplesReceived")
         setupDataProvider()
         //specify size of grid and layout
         projectGrid.setWidthFull()
-        //todo introduce variable for description column width
+        projectGrid.getColumn("ProjectTitle")
+                .setMinimumWidth(200)
```

Comment:
why do we have a minimum width here?

Reply:
because of 
```
projectGrid.getColumn("SamplesReceived")
     .setMaximumWidth(GridUtils.MAX_STATUS_COLUMN_WIDTH).setExpandRatio(1)
```
This causes Vaadin to grow this column. Without the MaxWidth of the samples received column the title column would disappear. Furthermore, I think that it is a good idea to have at least some characters shown to get an idea of the title.

Reply:
I see makes sense